R/looptools.R

```diff
@@ -155,7 +155,7 @@ loop_2pt <- function(loop_snk, loop_src,
     random_idcs <- expand.grid( 1:dims[1], 1:dims[1], KEEP.OUT.ATTRS = FALSE)
     colnames(random_idcs) <- c("r1", "r2")
     random_idcs <- dplyr::filter(random_idcs,
-                                 r1 != r2)
+                                 random_idcs$r1 != random_idcs$r2)
```

Comment:
I think I've found the correct and general solution for this using `rlang::.data`, will push in a few minutes.

Reply:
But really, `R CMD check` is outdated in this respect...

Reply:
To be honest though, I think I'll just replace this with a `which`